### PR TITLE
Issue/yamllint fixes

### PR DESCRIPTION
**Pull request by stvoutsin**
PR contains fixes to the YAML files based on the errors produced after running the Yamllint validator.
Changes were validated and tested via the yaml lint validation test, a manual set of notebook tests via the Zeppelin GUI, and the benchmarking tests.

### CODE REVIEW

**Comment by Zarquan** [comment]
I can see why changing `yes/no` to `true/false` is a good idea, but as long as they don't cause problems I'd like to keep the extra spaces after colon, by disabling the [`max-spaces-after`](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.colons) rule.

So this would stay as-is

    name:     "{{ deployname }}-{{ item }}"
    image:    "{{ hostvars[item]['image']  }}"
    flavor:   "{{ hostvars[item]['flavor'] }}"
    network:  "{{ networks['internal'] }}"
    
rather than being compressed into

    name: "{{ deployname }}-{{ item }}"
    image: "{{ hostvars[item]['image']  }}"
    flavor: "{{ hostvars[item]['flavor'] }}"
    network: "{{ networks['internal'] }}"

Purely aesthetic reasons, but as far as I can tell, so is limiting it to only one space.

**Comment by Zarquan** [comment]
This Stackoverflow comment clarifies how Ansible handles Yes/yes/True/true.
https://stackoverflow.com/questions/47877464/how-exactly-does-ansible-parse-boolean-variables

Turns out there is actually a standard [Boolean Language-Independent Type for YAML™ Version 1.1](https://yaml.org/type/bool.html), which actually allows all of the variants ` y|Y|yes|Yes|YES|n|N|no|No|NO |true|True|TRUE|false|False|FALSE |on|On|ON|off|Off|OFF`

File: deployments/hadoop-yarn/ansible/01-create-network.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@
         cloud: "{{ cloudname }}"
         state: present
         name: "{{ networks['internal'] }}"
-        shared:   'no'
+        shared: 'no'
         external: 'no'
```
Comment (Zarquan) [inline deployments/hadoop-yarn/ansible/01-create-network.yml:38]: What do we do about `'no'` as a quoted string.

Comment (stvoutsin) [reply deployments/hadoop-yarn/ansible/01-create-network.yml]: > This Stackoverflow comment clarifies how Ansible handles Yes/yes/True/true. https://stackoverflow.com/questions/47877464/how-exactly-does-ansible-parse-boolean-variables
> 
> Turns out there is actually a standard [Boolean Language-Independent Type for YAML™ Version 1.1](https://yaml.org/type/bool.html), which actually allows all of the variants ` y|Y|yes|Yes|YES|n|N|no|No|NO |true|True|TRUE|false|False|FALSE |on|On|ON|off|Off|OFF`

I've disabled the "truthy" rule, so the validation allows any of the values in that list, and changed the ansible scripts to remain as they were before.

**Comment by stvoutsin** [comment]
Latest commit should contain fixes based on your suggestions @Zarquan. The validation now allows any of the boolean variants, and allows empty spaces to allow the identation syntax we had previously. Note that the recent changes pass the validation.

File: deployments/hadoop-yarn/ansible/01-create-network.yml (hunk 1 above)
Comment (Zarquan) [reply deployments/hadoop-yarn/ansible/01-create-network.yml]: Sorry - my bad, not clear in the comments. I think the 'truthy' rule was a good idea, I was thinking aloud how we take this a step further and detect and correct `'yes'` values wrapped as strings.

Comment (Zarquan) [reply deployments/hadoop-yarn/ansible/01-create-network.yml]: Discussion so far:
- The 'truthy' rule standardising on `true` and `false` is a good idea
- If it causes no harm, I'd like to keep spaces after colon if we can, aesthetic reasons
- I'm volunteering to find and replace booleans wrapped in strings, `'yes'`, and correct them

Comment (stvoutsin) [reply deployments/hadoop-yarn/ansible/01-create-network.yml]: Ok I'll enable the "truthy" rule again, which will now complain about the boolean values that I've reverted back. Happy to leave the find and replace up to you, if it helps the last commit where made the changes back to the original values should give a better idea, but also the CI logs should show the files that need to be changed. I'll send a link after I make the change to the yamllint config file.

Comment (stvoutsin) [reply deployments/hadoop-yarn/ansible/01-create-network.yml]: Yamllint logs after reinstating the truthy rule: https://github.com/stvoutsin/aglais/runs/4792312023?check_suite_focus=true

Comment (stvoutsin) [reply deployments/hadoop-yarn/ansible/01-create-network.yml]: Actually one more clarification needed as I think I missread. Would you like me to replace all instances of yes/no (without quotes) with true/false for consistency as I did in the original set of changes, and you will replace the string versions (i.e. the ones wrapped in quotes) ?

Comment (stvoutsin) [reply deployments/hadoop-yarn/ansible/01-create-network.yml]: I've went ahead and changed all the yes/no values to true/false, and one instance of a "no" (in quotes) to false. Yamllint now passes the test without any warnings: https://github.com/stvoutsin/aglais/runs/4793633152?check_suite_focus=true
I haven't gone through the scripts to check but I assume there are a few instances of quotes booleans that the validator did not flag as a warning.

**Review by Zarquan — DISMISSED** [review]
A lot of careful work gone into that :+1:

**Review by Zarquan — APPROVED** [review]
:+1:

**Pull request merged**